torch/testing/_internal/opinfo/core.py

```diff
@@ -1769,7 +1769,9 @@ def generate_elementwise_binary_tensors(
     for shape in shapes:
         lhs = make_arg(shape, **op.lhs_make_tensor_kwargs)
         rhs = make_arg(shape, **op.rhs_make_tensor_kwargs)
-        yield SampleInput(lhs, args=(rhs,))
+        yield SampleInput(
+            lhs, args=(rhs,), kwargs=op.sample_kwargs(device, dtype, lhs)[0]
```

Comment:
Should we do a for-loop over all the sampled kwargs here? Why do we do [0] ?

Reply:
We do `[0]` because the tuple contains `(<args for torch>, <args for numpy>)` in the case of div with rounding mode floor, this tuple is: `({"rounding_mode": "floor"}, {"rounding_mode": "floor"})`.
It's the same as the implementation for some of the other Ops in common_method_invocations.py (threshold L16550, rrelu L16126, round 17219).